File: SRM.Domain/Data/UserData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SRM.Domain.Data
{
    public class UserData : ModelBase
    {
        public string? Username { get; set; }
        public string? Email { get; set; }
        public string? SoDienThoai { get; set; }
        public string? HoTen { get; set; }
        public string? MaVienChuc { get; set; }
        public string? GioiTinh { get; set; }
        public Guid? DonViId { get; set; }
        public string? TenDonVi { get; set; }
        public DateTime? NgaySinh { get; set; }
        public string? DanToc { get; set; }
        public string? ChucDanh { get; set; }
        public string? ChuyenNganh { get; set; }
        public string? HocVi { get; set; }
        public string? TruongDH { get; set; }
        public string? Role { get; set; }
    }
}

File: srm.client/src/models/UserData.js
import ModelBase from './ModelBase';

/**
 * @typedef {Object} UserData
 * @property {string} [id] - Unique identifier
 * @property {string} [ngayTao] - Creation date
 * @property {string} [ngaySua] - Modification date
 * @property {string} [username] - Username
 * @property {string} [email] - Email address
 * @property {string} [soDienThoai] - Phone number
 * @property {string} [hoTen] - Full name
 * @property {string} [maVienChuc] - Officer code
 * @property {string} [gioiTinh] - Gender
 * @property {string} [donViId] - Organization unit ID
 * @property {string} [tenDonVi] - Organization unit name
 * @property {string} [ngaySinh] - Birth date
 * @property {string} [danToc] - Ethnicity
 * @property {string} [chucDanh] - Title/Position
 * @property {string} [chuyenNganh] - Specialization
 * @property {string} [hocVi] - Academic degree
 * @property {string} [truongDH] - University
 * @property {string} [role] - User role
 */

/**
 * User data model
 */
const UserData = {
  ...ModelBase,
  username: null,
  email: null,
  soDienThoai: null,
  hoTen: null,
  maVienChuc: null,
  gioiTinh: null,
  donViId: null,
  tenDonVi: null,
  ngaySinh: null,
  danToc: null,
  chucDanh: null,
  chuyenNganh: null,
  hocVi: null,
  truongDH: null,
  role: null
};

export default UserData;